packages/saltcorn-data/models/file.js

```diff
@@ -97,7 +98,9 @@ class File {
    * @returns {string} - path to file
    */
   static get_new_path(suggest) {
-    const file_store = db.connectObj.file_store;
+    // Check if it uses S3, then use a default "saltcorn" folder
+    const useS3 = db.connectObj.storage_s3_enabled;
+    const file_store = useS3 ? db.connectObj.file_store : "saltcorn/";
```

Comment:
I think this should be `useS3 ? "saltcorn/" : db.connectObj.file_store;` ie swapped. otherwise the tests fail.

Reply:
Ah yes I think I mistyped this